### PR TITLE
S2vx web

### LINKED ISSUE #559: Style website
As a user, I want the website to be styled so it is more interesting and clearer to look at.

* Square themed
* Dark theme-ish
* Follow osu!'s example if we want: https://osu.ppy.sh/home

### PR DESCRIPTION
**Pull request by Naranja-Sagged**
Closes #559

### CODE REVIEW

**Comment by maxrchung** [comment]
Need to add label to pass validate pull request rule

File: S2VX.Web/Pages/Counter.razor [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+
+@code {
+    private int currentCount = 0;
+
+    private void IncrementCount()
+    {
+        currentCount++;
+    }
+}
```
Comment (maxrchung) [inline S2VX.Web/Pages/Counter.razor:16]: Remove this and other unused pages page

File: S2VX.Web/wwwroot/css/app.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    width: 100%;
+    z-index: 1000;
+}
+
+#blazor-error-ui .dismiss {
+    cursor: pointer;
+    position: absolute;
+    right: 0.75rem;
+    top: 0.5rem;
+}
```
Comment (maxrchung) [inline S2VX.Web/wwwroot/css/app.css:79]: Remove unused styling

**Review by maxrchung — CHANGES_REQUESTED** [review]

File: S2VX.Web/Pages/Counter.razor (hunk 1 above)
Comment (Naranja-Sagged) [reply S2VX.Web/Pages/Counter.razor]: Ok, I removed Counter.razor and FetchData.razor in the latest commit

**Review by maxrchung — APPROVED** [review]

**Pull request merged**